scripts/lint-notebooks.sh

```diff
@@ -15,38 +18,8 @@ IFS=$'\n'
 for fn in $notebooks; do
     echo "Linting $fn ..."
 
-    colab_line=$(grep "colab.research.google.com" "$fn")
-    colab_slug=$(echo "$colab_line" | sed -E "s/^.*$colab_badge\($colab_url_prefix(.*)\).*$/\1/")
-    if [ -n "$colab_line" ] && [ -z "$colab_slug" ]; then
-        echo "[ERROR] Colab link is improperly formatted:"
-        echo "[ERROR] $colab_line"
-        errors=$((errors + 1))
-    elif [ -n "$colab_line" ] && [[ "$colab_slug" != "$fn" ]]; then
-        echo "[ERROR] Colab path does not match notebook path: $colab_slug"
-        errors=$((errors + 1))
-    fi
-
-    kaggle_line=$(grep "kaggle.com/kernels/welcome" "$fn")
-    kaggle_slug=$(echo "$kaggle_line" | sed -E "s/^.*$kaggle_badge\($kaggle_url_prefix(.*)\).*$/\1/")
-    if [ -n "$kaggle_line" ] && [ -z "$kaggle_slug" ]; then
-        echo "[ERROR] Kaggle link is improperly formatted:"
-        echo "[ERROR] $kaggle_line"
-        errors=$((errors + 1))
-    elif [ -n "$kaggle_line" ] && [[ "$kaggle_slug" != "$fn" ]]; then
-        echo "[ERROR] Kaggle path does not match notebook path: $kaggle_slug"
-        errors=$((errors + 1))
-    fi
-
-    download_line=$(grep "Download Notebook" "$fn")
-    download_slug=$(echo "$download_line" | sed -E "s/^.*\"$download_url_prefix([^\"]*).\".*$/\1/")
-    if [ -n "$download_line" ] && [ -z "$download_slug" ]; then
-        echo "[ERROR] Download Notebook link is improperly formatted:"
-        echo "[ERROR] $download_line"
-        errors=$((errors + 1))
-    elif [ -n "$download_line" ] && [[ "$download_slug" != "$fn" ]]; then
-        echo "[ERROR] Download Notebook path does not match notebook path: $download_slug"
-        errors=$((errors + 1))
-    fi
+    # Link checking disabled - frontmatter will be auto-generated during conversion
+    # from notebook file paths, so no need to validate links in source notebooks
```

Comment:
Instead of mangling this script, let's just take it out of CI. There's no point in running a no-op script as a CI step, and we might as well keep the script around until we're comfortable with the changes (at which point we can rework it and put it back in CI if we have other linting rules we want to apply, or else delete it entirely).